src/main/java/com/minecolonies/core/colony/buildings/workerbuildings/BuildingConcreteMixer.java

```diff
@@ -154,6 +146,21 @@ public int getMaxBuildingLevel()
         return CONST_DEFAULT_MAX_BUILDING_LEVEL;
     }
 
+    /**
+     * Get the max amount of concrete placed at once.
+     *
+     * @return the number of concrete.
+     */
+    public int getMaxConcretePlaced()
+    {
+        int size = 0;
+        for (List<BlockPos> positions : waterPos.values())
+        {
+            size += positions.size();
```

Comment:
huh?

Reply:
The positions stored in the map contain the depth as the key, and the positions of the water at the given level.

By the way the code looks it's technically allowed to have several water streams at the same time, there's no practical limit on this.